tests/installation/await_install.pm

```diff
@@ -219,4 +225,12 @@ sub run {
     }
 }
 
+sub post_fail_hook {
```

Comment:
I'm not sure if we need to override postfail_hook. Linuxrc are included in the yast logs, but in case of crash, logs are gone as are stored in the tmpfs. As per our discussion, logs collection in case of the crash can be handled with the proposal mentioned here: https://progress.opensuse.org/issues/91287

Other than that it's a good idea and we won't waste time in case installer crashed. Good job!

Reply:
> I'm not sure if we need to override postfail_hook. Linuxrc are included in the yast logs, but in case of crash, logs are gone as are stored in the tmpfs. As per our discussion, logs collection in case of the crash can be handled with the proposal mentioned here: https://progress.opensuse.org/issues/91287
> 
you are right. i havent noticed. but the *-y2logs.tar.bz2 contains the linuxrc. i will delete the `post_fail_hook`

> Other than that it's a good idea and we won't waste time in case installer crashed. Good job!

Reply:
removed on 88459691d